### ISSUE TITLE
老铁，有微信吗，请教点问题

### ISSUE THREAD

**Issue by dingjiangang** [yangchengtest/keras-yolov3-kitti#1]
老铁，我的微信时13052288554，有些问题想请教一下谢谢了！！

**Comment by yangchengtest** [comment]
你有问题，直接这里问好了，微信还是算了。

**Comment by dingjiangang** [comment]
210/210 [==============================] - 649s 3s/step - loss: 33.6659 - val_loss: 33.9017
Epoch 41/50
210/210 [==============================] - 646s 3s/step - loss: 33.8016 - val_loss: 32.8996
Epoch 42/50
210/210 [==============================] - 646s 3s/step - loss: 33.5968 - val_loss: 33.4849
Epoch 43/50
210/210 [==============================] - 641s 3s/step - loss: 33.4823 - val_loss: 34.0187
Epoch 44/50
210/210 [==============================] - 637s 3s/step - loss: 33.4887 - val_loss: 33.4804
Epoch 45/50
210/210 [==============================] - 638s 3s/step - loss: 33.6443 - val_loss: 32.6732
Epoch 46/50
210/210 [==============================] - 646s 3s/step - loss: 33.6298 - val_loss: 33.3465
Epoch 47/50
210/210 [==============================] - 640s 3s/step - loss: 33.5977 - val_loss: 33.0172
Epoch 48/50
210/210 [==============================] - 637s 3s/step - loss: 33.5998 - val_loss: 34.2595
Epoch 49/50
210/210 [==============================] - 638s 3s/step - loss: 33.5135 - val_loss: 33.2451
Epoch 50/50
210/210 [==============================] - 639s 3s/step - loss: 33.4252 - val_loss: 32.6822
Unfreeze all of the layers.
Train on 6733 samples, val on 748 samples, with batch size 4.
Epoch 51/100
1683/1683 [==============================] - 722s 429ms/step - loss: 32.1092 - val_loss: 30.0148
Epoch 52/100
1683/1683 [==============================] - 701s 417ms/step - loss: 29.7340 - val_loss: 28.4527
这样的训练kitti数据集正常吗？谢谢

**Comment by yangchengtest** [comment]
正常的，你拿实际MODEL去跑一下你的目标图片看一下效果。

**Comment by dingjiangang** [comment]
谢谢

**Comment by dingjiangang** [comment]
老铁，我训练完了，但是效果不是很好，还没有原版的好，有什么改进方式吗

**Comment by yangchengtest** [comment]
KITTI的图片分辨率和VOC等的分辨率不一样。
测试图如果和KITTI分辨率不一致，效果不好正常。
改的话，可以改ANCHORS，和分类数目。我个人觉得图片分辨率的影响更大。
